summary_workflows/quantification/quantification_dockers/apaeval_consolidation/manage_assessment_data.py

```diff
@@ -9,7 +9,7 @@
 import numpy as np
 from assessment_chart import assessment_chart
 
-DEFAULT_eventMark = '2018-04-05'
+DEFAULT_eventMark = '2021-06-04'
```

Comment:
Uuuum, I was thinking about defining it in `nextflow.config` and passing it as a parameter to `manage_assessment_data.py` in [main.nf](https://github.com/iRNA-COSI/APAeval/blob/add_working_Q2_summaryworkflow/summary_workflows/quantification/main.nf#L151) rather than creating a new python function that we need to update separately. Do you think this is a better option?

Reply:
That's a good idea! The `nextflow.config` now has a parameter called `eventMark`, which is passed to `manage_assessment_data.py` through an argument. The `docker` container is also updated and tested.